articles/configure-your-inputs.mdx

```diff
@@ -151,45 +151,45 @@ These selector-style keys can be as targeted as you need them to be:
 Each entry in `_inputs` has the following options available:
 
 <comp.DataReference>
- <comp.DataReferenceRow label="type" type_markdown="String">
+  <comp.DataReferenceRow label="type" type_markdown="String">
     Changes the editing interface for an input. Each input appears and behave differently, takes different `options`, and processes different values. See more details in the [type section](#type) below.
 
     For unknown types, or if the input contains an unsupported value (e.g. an array value in a text input), the `disabled` type is used instead.
- </comp.DataReferenceRow>
+  </comp.DataReferenceRow>
   <comp.DataReferenceRow label="label" type_markdown="String">
     Changes the text above an input. Defaults to generating the label from the input key (e.g. "Page Title" for `page_title`).
- </comp.DataReferenceRow>
+  </comp.DataReferenceRow>
   <comp.DataReferenceRow label="comment" type_markdown="String">
     Changes the subtext below the `label`. Has no default. Supports a limited set of Markdown: links, bold, italic, subscript, superscript and inline code elements are allowed.
 
     Useful for adding reminders or additional context for your team members on a specific input.
- </comp.DataReferenceRow>
+  </comp.DataReferenceRow>
   <comp.DataReferenceRow label="options" type_markdown="Object">
     Configuration settings passed on to the input when displayed. These are specific to each input. The [available types and their options](#types-reference) are listed below.
- </comp.DataReferenceRow>
+  </comp.DataReferenceRow>
   <comp.DataReferenceRow label="instance_value" type_markdown="String">
     Controls if and how the value of this input is instantiated when created. This occurs when creating files, or adding array items containing the configured input. Has no default. Must be one of `UUID` or `NOW`.
 
     * `UUID` generates a uuidv4 ([extremely unlikely to generate duplicates)](https://softwareengineering.stackexchange.com/a/130298), useful for identifying unique items (e.g. `6ec0bd7f-11c0-43da-975e-2a8ad9ebae0b`)
     * `NOW` generates the current datetime in the site's configured timezone
- </comp.DataReferenceRow>
+  </comp.DataReferenceRow>
   <comp.DataReferenceRow label="hidden" type_markdown="Boolean or string">
     Toggles the visibility of this input. Useful for when you need an input to exist, but don't want it edited or seen. Defaults to the naming convention, `true` for input keys starting with an underscore and `false` otherwise.
 
     If set as a boolean, the input is hidden based if `true`.
 
-    If set as a string, the input is hidden based on the value of another input. This can start with a `!` to reverse the value.
+    If set as a string, the input is hidden based on the value of another input. This can start with a `!` to reverse the value. For reverse values in `YAML` files, the string should be in `'` of `"` quotation marks.
```

Comment:
```suggestion
    If set as a string, the input is hidden based on the value of another input. This can start with a `!` to reverse the value. For reverse values in `YAML` files, the string should be in `'` or `"` quotation marks.
```